src/components/User/Assets/AssetRequests/AssetRequests.Show.vue

```diff
@@ -266,9 +266,9 @@ export default {
   computed: {
     isCancellable () {
       const isPending =
-        this.tokenRequest.state === 'pending'
+        this.assetRequest.state === 'pending'
```

Comment:
Create some constant for `pending` state. And replace this string to the constant value through component